solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -25,20 +25,78 @@
 import org.apache.lucene.search.Explanation;
 import org.apache.solr.ltr.feature.Feature;
 import org.apache.solr.ltr.norm.Normalizer;
-import org.apache.solr.ltr.util.LTRUtils;
+import org.apache.solr.util.SolrPluginUtils;
 
 public class LambdaMARTModel extends LTRScoringModel {
 
-  private final List<RegressionTree> trees = new ArrayList<RegressionTree>();
+  private final HashMap<String,Integer> fname2index;
```

Comment:
Do we need so many references to this throughout the code? Inner classes can access their enclosing class's variables and in this case I think it would make the code cleaner.